libs/utils/utils.js

```diff
@@ -461,6 +502,8 @@ export function decorateLinks(el) {
     }
     return rdx;
   }, []);
+  await decorateLinksToButtons(anchors);
```

Comment:
This doesn't seem right to me. We're just iterating over these links again. We should be checking the link in the reduce.

Reply:
makes sense, I'll move the call within the reduce. 👍